### PR TITLE
fix #3363: Field length with 0 value.

### LINKED ISSUE #3363: [Bug Report] Length in 0 of column, prevents to register values in that column.
#### Steps to reproduce
1. Open Table window.
2. Add column.
3. Select `Amt` in the system element to set the reference `Amount`.

#### Screenshot or Gif
amt_error

#### Expected behavior
The column length that has been set is expected to maintain the value when the record is saved.
Because if the length is kept at 0, it does not record the values in that column.

#### Other relevant information
- Your OS: Linux Mint 20.1 x64. 
- Java version: 1.8.0_281.
- ADempiere version: 3.93 update to develop c91b62d.
- DB Provider: PostgreSQL.
- DB version: 12.5.

#### Additional context
The references to which this behaviour applies:
* `Amount`.
* `Binary`.
* `Cost+Prices`.
* `Date`.
* `Date+Time`.
* `Number`.
* `Quantity`.
* `Time`.

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
field-lengrh-fixed

fixes #3363

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "editor.trimAutoWhitespace": false,
+    "diffEditor.ignoreTrimWhitespace": false,
+    "editor.find.addExtraSpaceOnTop": false,
```
Comment (yamelsenih) [inline .vscode/settings.json:4]: Please remove this file, is unnecessary

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (EdwinBetanc0urt) [reply .vscode/settings.json]: the vs code ide configuration file has been deleted.

**Comment by yamelsenih** [comment]
Testing

File: base/src/org/compiere/model/MColumn.java [hunk 2]
```diff
@@ -198,13 +201,7 @@ public static String getColumnName (Properties ctx, int AD_Column_ID)
 	public MColumn (Properties ctx, int AD_Column_ID, String trxName)
 	{
 		super (ctx, AD_Column_ID, trxName);
-		if (AD_Column_ID == 0)
```
Comment (yamelsenih) [inline base/src/org/compiere/model/MColumn.java:201]: Please do not merge little changes with functionality change, remove this

File: base/src/org/compiere/model/MColumn.java [hunk 3]
```diff
@@ -328,7 +325,7 @@ public void setIsEncrypted (boolean IsEncrypted)
 	protected boolean beforeSave (boolean newRecord)
 	{
 		//set column default based in element when is a new column FR [ 3426134 ]
-		if(newRecord) {
```
Comment (yamelsenih) [inline base/src/org/compiere/model/MColumn.java:331]: Same here

**Review by yamelsenih — CHANGES_REQUESTED** [review]

**Pull request merged**